### PR TITLE
Feature/responsive design ect

### PR DESCRIPTION
**Pull request by ectweitmann**
## What Features are Being Added?
- Responsive design for S/M/L phones, tablets, and laptops/desktops
- Lots of media queries

## What is Being Refactored?
- Tweak default stylings of elements when/where necessary

## Where should the reviewer start?
- Try out the app in different layouts.

## How should this be manually tested?
- Open up the dev tools and try out the different pages of the app with different device simulators

## What are the relevant tickets?
- Responsive design

## Any Other Notes/Questions/Problems?
- Tried to keep my commits lean and mean haha. Everything is responsive to different screen sizes. It proved especially difficult to do, so there may be a few mishaps for some views under certain screen widths, but I tried my best to get everything looking good in every page view on every device.

### CODE REVIEW

**Comment by cagallo** [comment]
- Looks great Ethan!

File: src/styles/BookCard.css [hunk 1]
```diff
@@ -62,3 +62,13 @@
   background: #938673;
   color: #FBF3EE;
 }
+
+@media screen and (max-width: 320px) {
+  .book-title {
```
Comment (krdurrett) [inline src/styles/BookCard.css:67]: This makes sense for small screens

File: src/styles/BookContainer.css [hunk 2]
```diff
@@ -23,3 +23,34 @@
...
+@media screen and (max-width: 500px) {
+  .book-container {
+    grid-template-columns: repeat(auto-fit, minmax(100px, 84%));
+    grid-auto-rows: minmax(155px, 320px);
+    grid-row-gap: 12%;
+    grid-column-gap: 4.9%;
+    padding-bottom: 7.6%;
+  }
+}
+
```
Comment (krdurrett) [inline src/styles/BookContainer.css:36]: Strong work on these media queries!

File: src/styles/BookDetails.css [hunk 3]
```diff
@@ -85,3 +92,91 @@
...
+}
+
+@media screen and (min-width: 330px) and (max-width: 500px) {
+  .selected-cover {
+    width: 150px;
+  }
+}
+
+@media screen and (min-width: 501px) and (max-width: 1023px) {
+  .selected-cover {
```
Comment (krdurrett) [inline src/styles/BookDetails.css:161]: So much adjustment needed! Thanks Ethan!

**Review by krdurrett — APPROVED** [review]
This work is so tedious, thanks for tackling that

File: src/styles/BookDetails.css [hunk 4]
```diff
@@ -85,3 +92,91 @@
   background: #938673;
   color: #fbf3ee;
 }
+
+.book-details::-webkit-scrollbar {
+  display: none;
+}
+
+@media screen and (max-width: 1023px) {
```
Comment (Fordo29) [inline src/styles/BookDetails.css:100]: Thank you for all your work on this Ethan.

File: src/styles/Nav.css [hunk 5]
```diff
@@ -15,10 +15,33 @@
...
+  width: 40%;
+  max-width: 250px;
 
 }
 
 .logo {
   width: 100%;
 }
+
+@media screen and (max-width: 744px) {
```
Comment (Fordo29) [inline src/styles/Nav.css:27]: This is neat to see how you handled the nav bar in styling

**Pull request merged**